variables.tf

```diff
@@ -290,11 +290,11 @@ variable "launch_template_version" {
 
 variable "resources_to_tag" {
   type        = list(string)
-  description = "List of auto-launched resource types to tag. Valid types are \"instance\", \"volume\", \"elastic-gpu\", \"spot-instances-request\"."
+  description = "List of auto-launched resource types to tag. Valid types are \"instance\", \"volume\", \"elastic-gpu\", \"spot-instances-request\", \"network-interface\"."
   default     = []
   validation {
     condition = (
-      length(compact([for r in var.resources_to_tag : r if !contains(["instance", "volume", "elastic-gpu", "spot-instances-request"], r)])) == 0
+      length(compact([for r in var.resources_to_tag : r if !contains(["instance", "volume", "elastic-gpu", "spot-instances-request", "network-interface"], r)])) == 0
     )
     error_message = "Invalid resource type in `resources_to_tag`. Valid types are \"instance\", \"volume\", \"elastic-gpu\", \"spot-instances-request\"."
```

Comment:
add `"network-interface"` to `error_message`

Reply:
Fixed

Reply:
Good catch @aknysh -- Thanks 👍